packages/slice-machine/lib/models/common/widgets/Image/components.tsx

```diff
@@ -75,84 +78,103 @@ export const AddThumbnailButton = ({ onClick }) => (
+            ? null
+            : cast
+          : cast;
+      helpers.setValue({ ...field.value, [key]: value });
+    };
 
   return (
     <Fragment>
       <FormFieldInput
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/restrict-template-expressions
         fieldName={`${prefix}.name`}
         meta={{
           ...meta,
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-assignment
-          error: meta.error && meta.error.name,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:disable
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+          error: meta.error && meta.error?.name,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:enable
         }}
         formField={{ label: `Name${requiredChar}`, placeholder: "main" }}
         field={
           prefix === "config.constraint"
-            ? { value: "main", readOnly: true }
-            : // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/restrict-template-expressions, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
+            ? // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+              // @ts-ignore:disable
+              { value: "main", readOnly: true }
+            : // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
               { value: field.value.name, onChange: createSetField("name") }
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:enable
         }
         variant={prefix === "config.constraint" ? "disabled" : "primary"}
         sx={{ mb: 3 }}
       />
       <FormFieldInput
-        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
         fieldName={`${prefix}.width`}
         meta={{
           ...meta,
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/restrict-template-expressions, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:disable
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
           error: meta.error && meta.error.width,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:enable
         }}
         formField={{ label: `Width (px)${requiredChar}`, placeholder: " " }}
         field={{
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:disable
           type: "number",
```

Comment:
Pass this to `fieldProps` to not conflict with TS field type

Reply:
found a better way :)